hnn_core/gui/_viz_manager.py

```diff
@@ -87,6 +108,54 @@ def _figname2idx(fname):
     return int(fname.split(" ")[-1])
 
 
+def _calculate_rmse(dpl, dpl_target, tstart, tstop, weights=None):
```

Comment:
don't we have a function for this already?

https://github.com/jonescompneurolab/hnn-core/blob/cdd4050f187da4bc84c2869952a8fd54ae1e70dd/hnn_core/dipole.py#L172

Reply:
oh no... I didn't realize that